lib/pattern-preset.js

```diff
@@ -77,6 +77,11 @@ export const presets = {
     ],
   },
 
+  RequirementsTxt: {
+    pathSubstrings: ['requirements.txt'],
+    githubClasses: [],
```

Comment:
I don't think this will be an issue. It will just invoke the plugin and if there is no match with the regex it will not harm the user. Does it make sense to you?